apps/gro/controllers/no-postcode.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const controllers = require('hof').controllers;
+const BaseController = controllers.base;
+
+module.exports = class NoPostcodeController extends BaseController {
+  locals(req, res, callback) {
+    const locals = super.locals(req, res, callback);
```

Comment:
`locals` is defined in the outer scope as the function name. Can you rename the `const`?